zeebe/pom.xml

```diff
@@ -61,4 +60,20 @@
     <module>restore</module>
     <module>topology</module>
   </modules>
+
+  <profiles>
+    <!-- 
```

Comment:
🔧 Following up on the [discussion](https://github.com/camunda/zeebe/pull/17497/files#r1570100625) I would suggest the following to make clear we want to ignore the whole module not just skip test runs.
```suggestion
    <!-- 
      The QA module is enabled per default, but having it in a profile allows us to exclude it easily.
      That is useful for example for creating releases.
       -->
```